birdhouse/components/monitoring/prometheus.rules.template

```diff
@@ -93,71 +93,71 @@ groups:
 
 
 
   - alert: HostUnusualDiskWriteLatency
-    expr: rate(node_disk_write_time_seconds_total[2m]) / rate(node_disk_writes_completed_total[2m]) > 100
+    expr: rate(node_disk_write_time_seconds_total[2m]) / rate(node_disk_writes_completed_total[2m]) > $PROMETHEUS_HostUnusualDiskWriteLatency_ALERT
     for: 5m
     labels:
       severity: warning
     annotations:
       summary: "Host unusual disk write latency (instance {{ $labels.instance }})"
-      description: "Disk latency is growing (write operations > 100ms)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+      description: "Disk latency is growing (write operations > ${PROMETHEUS_HostUnusualDiskWriteLatency_ALERT}ms)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
 
 
 
   - alert: HostHighCpuLoad
-    expr: 100 - (avg by(instance) (irate(node_cpu_seconds_total{mode="idle"}[5m])) * 100) > 80
+    expr: 100 - (avg by(instance) (irate(node_cpu_seconds_total{mode="idle"}[5m])) * 100) > $PROMETHEUS_HostHighCpuLoad_ALERT
     for: 5m
     labels:
       severity: warning
     annotations:
       summary: "Host high CPU load (instance {{ $labels.instance }})"
-      description: "CPU load is > 80%\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+      description: "CPU load is > ${PROMETHEUS_HostHighCpuLoad_ALERT}%\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
 
 
 
-  # 1000 context switches is an arbitrary number.
+  # Number of context switches is an arbitrary number.
   # Alert threshold depends on nature of application.
   # Please read: https://github.com/samber/awesome-prometheus-alerts/issues/58
   - alert: HostContextSwitching
-    expr: (rate(node_context_switches_total[5m])) / (count without(cpu, mode) (node_cpu_seconds_total{mode="idle"})) > 2000
+    expr: (rate(node_context_switches_total[5m])) / (count without(cpu, mode) (node_cpu_seconds_total{mode="idle"})) > $PROMETHEUS_HostContextSwitching_ALERT
     for: 5m
     labels:
       severity: warning
     annotations:
       summary: "Host context switching (instance {{ $labels.instance }})"
-      description: "Context switching is growing on node (> 1000 / s)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+      description: "Context switching is growing on node (> $PROMETHEUS_HostContextSwitching_ALERT / s)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
 
 
 
   - alert: HostSwapIsFillingUp
-    expr: (1 - (node_memory_SwapFree_bytes / node_memory_SwapTotal_bytes)) * 100 > 80
+    expr: (1 - (node_memory_SwapFree_bytes / node_memory_SwapTotal_bytes)) * 100 > $PROMETHEUS_HostSwapIsFillingUp_ALERT
     for: 5m
     labels:
       severity: warning
     annotations:
       summary: "Host swap is filling up (instance {{ $labels.instance }})"
-      description: "Swap is filling up (>80%)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+      description: "Swap is filling up (> $PROMETHEUS_HostSwapIsFillingUp_ALERT %)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
```

Comment:
unnecessary space before the `%`

Reply:
I inserted the space on purpose so I do not have to surround `PROMETHEUS_HostSwapIsFillingUp_ALERT` with `{}`.  Same reason for all the extra spaces inserted elsewhere.